roles/bind-keycloak-apb/tasks/main.yml

```diff
@@ -7,22 +7,26 @@
   register: generated_client_secret
 
 - name: Retrieve route to keycloak
-  shell: "oc get configmaps/{{keycloak_service_name}} -n {{ namespace }} -o jsonpath={.data.uri}"
+  shell: "oc get configmaps/{{ keycloak_service_name }} -n {{ namespace }} -o jsonpath={.data.uri}"
   register: keycloak_configmap_uri
 
-- name: Retrieve {{ _apb_provision_creds.USERNAME }} user realm
-  shell: "oc get configmaps/{{keycloak_service_name}} -n {{ namespace }} -o jsonpath={.data.realm}"
+- name: Retrieve keycloak realm
```

Comment:
Shold not it be `{{ _apb_provision_creds.USERNAME }}` user realm? We have custom user who should create bindings and public client and its bearer should be created